SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -32,6 +34,84 @@ void USpatialPlayerSpawner::Init(USpatialNetDriver* InNetDriver)
 	NetDriver = InNetDriver;
 }
 
+void USpatialPlayerSpawner::Advance(const TArray<Worker_Op>& Ops)
+{
+	QueryHandler.ProcessOps(Ops);
+
+	for (const Worker_Op& Op : Ops)
+	{
+		if (Op.op_type == WORKER_OP_TYPE_COMMAND_REQUEST)
+		{
+			const Worker_CommandRequestOp& CommandRequestOp = Op.op.command_request;
+			const Worker_CommandRequest& Request = CommandRequestOp.request;
+			const Worker_EntityId EntityId = CommandRequestOp.entity_id;
+			const Worker_ComponentId ComponentId = Request.component_id;
+			const Worker_RequestId RequestId = CommandRequestOp.request_id;
+			const Schema_FieldId CommandIndex = Request.command_index;
+
+			if (ComponentId == SpatialConstants::PLAYER_SPAWNER_COMPONENT_ID
+				&& CommandIndex == SpatialConstants::PLAYER_SPAWNER_SPAWN_PLAYER_COMMAND_ID)
+			{
+				NetDriver->PlayerSpawner->ReceivePlayerSpawnRequestOnServer(CommandRequestOp);
```

Comment:
Few references to the PlayerSpawner via the net driver, which AFAIK will always be this object